cdap-docs/faqs/source/cdap.rst

```diff
@@ -83,8 +83,8 @@ I've followed the install instructions, yet CDAP does not start and fails verifi
 If you have followed :ref:`the installation instructions <installation-index>`, and CDAP either did not pass the 
 :ref:`verification step <admin-manual-verification>` or did not startup, check:
 
-- Look in the CDAP logs for error messages (located either in ``/var/log/cdap`` for CDAP Distributed or 
-  ``$CDAP_HOME/logs`` for CDAP SDK Standalone)
+- Look in the CDAP logs for error messages (located either in ``/var/log/cdap`` for Distributed CDAP or 
+  ``$CDAP_HOME/logs`` for Standalone CDAP from the CDAP SDK)
```

Comment:
too many CDAPs? (although that seems like a good problem to have generally, perhaps not here? :-))